README.md

```diff
@@ -1,5 +1,4 @@
 # ofCourse - Open, Free Course(ware)
-
```

Comment:
Some markdown parsers require a blank line after a # header, please put that back.